configure.ac

```diff
@@ -1990,18 +1996,18 @@ AS_IF([test x"$zstd_status" = "xok"],
 
 ## Determine whether the debugger should/can be built
 
-AS_CASE([$enable_debugger],
+AS_CASE([$enable_ocamldebug],
   [no],
     [with_debugger=""
     build_ocamldebug=false
-    AC_MSG_NOTICE([replay debugger disabled])],
+    AC_MSG_NOTICE([ocamldebug disabled])],
   [AS_IF([$sockets],
-    [with_debugger="ocamldebugger"
+    [with_debugger="ocamldebug"
```

Comment:
If you change this, you have to change `Makefile` and `ocamltest/Makefile` because the `ocamldebugger`  value is significant.

Reply:
Damn, I thought that I had checked this. I don't see a reason to use the name "ocamldebugger", I guess that either "ocamldebug" or "debugger" make more sense, so I'll go ahead and change `./Makefile`. What do I need to change in `ocamltest/Makefile`? I don't see any occurrence of `ocamldebugger` in the `ocamltest/` subdirectory.

Reply:
> What do I need to change in `ocamltest/Makefile`?

Nothing. I can't find it in trunk, only in 4.14, which I was probably looking at when I wrote that.

Reply:
(So I assume that this is now fine and that you may be planning to approve this PR?)